news/1529.feature

```diff
@@ -0,0 +1 @@
+Add `env` to Hydra's config group
```

Comment:
Let's keep this one internal. Without documentation this means absolutely nothing to users and I don't think we should document it right now.